### PR TITLE
clustering: Give time to minio container to come up

### PR DESCRIPTION
**Pull request by peusebiu**
**What type of PR is this?**

**Which issue does this PR fix**:

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by rchincha** [comment]
can we not do a HTTP health check instead?

**Comment by peusebiu** [comment]
> can we not do a HTTP health check instead?
> 
> --name=minio --health-cmd "curl http://localhost:9000/minio/health/live"?

we have one, but it needs to be inspected with a script or something which waits until is healthy

File: .github/workflows/cluster.yaml [hunk 1]
```diff
@@ -70,6 +70,7 @@ jobs:
         run: pip3 install minio
       - name: Create minio bucket
         run: |
+            sleep 30
```
Comment (rchincha) [inline .github/workflows/cluster.yaml:73]: Instead of this sleep, can we do curl health checks until ready.

**Review by rchincha — APPROVED** [review]
lgtm

**Pull request merged**